pkg/api/login.go

```diff
@@ -210,7 +210,21 @@ func (hs *HTTPServer) LoginPost(c *models.ReqContext) response.Response {
 	err := bus.Dispatch(c.Req.Context(), authQuery)
 	authModule = authQuery.AuthModule
 	if err != nil {
-		resp = response.Error(401, "Invalid username or password", err)
+		var msg string
+
+		if errors.Is(err, login.ErrTooManyLoginAttempts) {
+			msg = "Too many consecutive incorrect login attempts for user - login for user temporarily blocked"
```

Comment:
alternatively, you could do the following in order to use [this text](https://github.com/grafana/grafana/blob/77347a37b1741a592c46cf5ff7cd5f8d812795b3/pkg/login/auth.go#L18).
```suggestion
			msg = err.Error()
```

You can also apply the same for the other cases.